CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# www.jsdelivr.com Contributing Guide
+
+Hi! We're really excited that you're interested in contributing to www.jsdelivr.com! Before submitting your contribution, please read through the following guide.
+
+## Overview
+
+The website uses [Ractive.js](https://ractive.js.org/), [ractive-route](https://github.com/MartinKolarik/ractive-route), and a custom build of [Bootstrap v3](https://getbootstrap.com/). To add a new page, you need to create a new `.html` in [views/pages](https://github.com/jsdelivr/www.jsdelivr.com/tree/master/src/views/pages) (use one of the existing ones as a reference), and add it to [client-side routing](https://github.com/jsdelivr/www.jsdelivr.com/blob/master/src/public/js/app.js). All `.html` files are compiled as [Ractive.js components](https://ractive.js.org/api/#component-files). Styles are in a separate [less directory](https://github.com/jsdelivr/www.jsdelivr.com/tree/master/src/public/less), which mirrors the `views` structure.
+
+### Guidelines
+
+ - Bug fixes and changes discussed in the existing issues are always welcome.
+ - For new ideas, please open an issue to discuss them before sending a PR.
+ - Make sure your PR passes `npm test` and has [appropriate commit messages](https://github.com/jsdelivr/www.jsdelivr.com/commits/master).
+ - A Heroku preview is built automatically for every PR and will be reviewed before merging.
```

Comment:
Maybe just a "A preview instance" without mentioning the provider

Reply:
There's no instance at all for outside contributor PRs.

Reply:
render docs dont seem to mention this, they make it seem that any PR against master will create a preview

Reply:
🤷‍♂️